changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "3.0.3": [
+      "[Added] Add Aptana Studio support - #14669. Thanks @tsvetilian-ty!",
+      "[Fixed] Fix crash when user's locale is unsupported by the spellchecker - #14817. Thanks @tsvetilian-ty!",
+      "[Fixed] On Windows, remember the app's zoom level on update - #5315",
+      "[Fixed] Fix Markdown syntax highlighting - #14710",
```

Comment:
```suggestion
```

Did this change user facing behavior? or just console log.  If it does change user facing behavior, maybe we can be more exact.

Reply:
Yeah, before this markdown files weren't highlighted at all 😕 
Before:
![image](https://user-images.githubusercontent.com/1083228/171396812-b52c4689-1ec2-440a-b320-352359673b1a.png)

After:
![image](https://user-images.githubusercontent.com/1083228/171396736-ed227062-014a-4339-9d62-1dd5462023f7.png)

Reply:
Oh ok. I guess my confusion was that with the word "syntax" I thought the diff highlighting was not working not the auto selection no working/highlighting for selection.

Maybe it should "Fix markdown selection highlighting"? or "Fix changes not being auto selected in markdown files"